lib/src/html_generator.dart

```diff
@@ -185,7 +185,7 @@ class HtmlGeneratorInstance {
   }
 
   void _generateSearchIndex() {
-    File jsonFile = createOutputFile(out, 'index.json');
```

Comment:
One downside is that we're preventing potential interesting uses of our docs by not publishing a JSON file. I'm tempted to continue to generate this file, in order to allow other tools to experiment.